src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -36,22 +36,53 @@ Removing or adding an authorized network on your CockroachDB {{ site.data.produc
+{{site.data.alerts.end}}
 
 Azure Private Link is not yet available for [CockroachDB {{ site.data.products.dedicated }} on Azure]({% link cockroachcloud/cockroachdb-dedicated-on-azure.md %}).
 
 {{site.data.alerts.callout_info}}
 {% include cockroachcloud/cdc/kafka-vpc-limitation.md %}
 {{site.data.alerts.end}}
 
+#### Private Service Connect
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+1. Navigate to your cluster's **Networking > Private endpoint** tab.
+1. Click **Add connection**. Copy the value provided for **Target service**. Do not close this browser window.
+1. In a new browser window, log in to Google Cloud Console, go to **Private Service Connect** section, and create a new endpoint in the same VPC as your application. For details, refer to [Create an endpoint](https://cloud.google.com/vpc/docs/configure-private-service-connect-services#create-endpoint) in the Google Cloud documentation.
+    - Set **Target** to **Published service**.
+    - Set **Target service** to the value you copied from CockroachDB {{ site.data.products.cloud }} Console. If the endpoint's configured target service does not match, validation will fail.
+    - Provide a value for **Endpoint name**. This is not used by CockroachDB {{ site.data.products.cloud }}.
+    - If it is not enabled, enable the Service Directory API, click **Enable global access*, and create a namespace in each region where your cluster is deployed.
+    - Click **Add endpoint**.
+    - After the endpoint is created, copy the connection ID.
+1. Return to the CockroachDB {{ site.data.products.cloud }} Console browser tab and click **Next**.
+1. Enter the endpoint's ID, then click **Validate**. CockroachDB {{ site.data.products.cloud }} attempts to connect to the endpoint's VPC and verifies that the target service matches the cluster. If validation fails, verify the endpoint's configuration, then try again. After validation succeeds, click **Next** to finish creating the connection.
+1. On the **Networking > Private endpoint** tab, verify that the connection status is **Available**.
+
+{{site.data.alerts.callout_success}}
+After validation succeeds for an endpoint, additional endpoints in the same VPC are automatically automatically accepted if they are configured with the cluster's target service ID. Additional VPCs must be added separately.
+{{site.data.alerts.end}}
+
+##### Known limitations
+
+Deleting a PSC connection from CockroachDB {{ site.data.products.cloud }} is not supported. Instead, remove or disable the connection from GCP or modify its target service ID so that it no longer matches the cluster. When a connection no longer exists, CockroachDB {{ site.data.products.cloud }} automatically removes it from the cluster.
+
 #### VPC Peering
```

Comment:
Shouldn't VPC peering be one heading highere? ### VPC Peering

Reply:
It is at level 4, the same level as PSC.